### PR TITLE
add mutagen lab module

### PR DESCRIPTION
**Pull request by curstwist**
#### Summary
Content "New lab modular mutagen area."

#### Purpose of change
Continue adding content for modular labs.

EDIT: dependent on lab modular residential areas #50189 for new trap json.
#### Describe the solution
This is a 2 level mutagen research area.  I have added a variety of vault types for mutagens and serums.  The serums will eventually more to a more secure facility and this will become a beginners/level 1 mutagen lab map.

#### Describe alternatives you've considered

Not adding serums, but it will take a while to make the more secure facility so I wanted to offer them here as a nested map chance.

This also introduces "biome" areas for researchers, which will provide a place to begin adding content for alien environments.  I'd added some basic breather home world terrain/flora as a rare nest and updated the mi-go containment area significantly.  Alien biomes may move to a portal themed lab later (TBD).

This lab focuses on animal based research.
#### Testing

Play testing 

#### Additional context

<details>
  <summary>Spoiler warning</summary>
  
upper tier:
image

lower tier:  I've noticed that the biome nests are fond of spawning the same nest in each location, I'm not sure why.
image

</details>

### CODE REVIEW

File: data/json/furniture_and_terrain/furniture-alien.json [hunk 1]
```diff
@@ -621,5 +621,18 @@
         { "item": "exodii_computer", "count": [ 0, 1 ] }
       ]
     }
+  },
+  {
+    "type": "furniture",
+    "id": "f_crystaline_flower",
+    "name": "crystaline flower",
+    "description": "A crystaline structure that resembles a plant.",
```
Comment (actual-nh) [inline data/json/furniture_and_terrain/furniture-alien.json:629]:
```suggestion
    "id": "f_crystalline_flower",
    "name": "crystalline flower",
    "description": "A crystalline structure that resembles a plant.",
```

File: data/json/mapgen/lab/lab_modular/lab_nests_modular/lab_nested_mutagen.json [hunk 2]
```diff
@@ -332,6 +434,40 @@
...
+        "....6d.....d....",
+        "................",
+        "................",
+        ".........d......",
+        "..............6.",
+        "................"
+      ],
+      "place_nested": [ { "chunks": [ [ "lab_16x1_bash", 80 ], [ "null", 20 ] ], "x": 0, "y": 0 } ],
+      "terrain": { ".": "t_gelatinous_groundcover", "6": "t_gelatinous_groundcover", "|": "t_concrete_wall", "X": "t_laminated_glass" },
+      "furniture": { "d": "f_crystaline_flower", "6": "f_fog_breather" },
```
Comment (actual-nh) [inline data/json/mapgen/lab/lab_modular/lab_nests_modular/lab_nested_mutagen.json:461]:
```suggestion
      "furniture": { "d": "f_crystalline_flower", "6": "f_fog_breather" },
```

**Review by actual-nh — COMMENTED** [review]
Spelling correction, including in id.

**Comment by Zireael07** [comment]
Biome areas look cool <3

**Pull request merged**